db/knex_init_db.js

```diff
@@ -548,6 +548,20 @@ ALTER TABLE monitor
         table.double("timeout").defaultTo(0).notNullable();
     });
 
+    /*
+    patch-add-ping-advanced-options.sql
+    ALTER TABLE monitor ADD ping_count INTEGER default 1 not null;
+    ALTER TABLE monitor ADD ping_numeric BOOLEAN default true not null;
+    ALTER TABLE monitor ADD ping_deadline INTEGER default 10 not null;
+    ALTER TABLE monitor ADD ping_timeout INTEGER default 2 not null;
+    */
+    await knex.schema.table("monitor", function (table) {
```

Comment:
please remove this part and add it to a [new migration](https://github.com/louislam/uptime-kuma/tree/003ffc2cc2a3b572c197658b08b34a6618f804fa/db/knex_migrations), and only there (the `.sql` files are a crutch which we did not have the time/energy to clean up yet)

I think this file will only be run for new users, but not existing users.